pkg/apis/experimental/types.go

```diff
@@ -464,9 +464,9 @@ type JobCondition struct {
 	Message string `json:"message,omitempty"`
 }
 
-// An Ingress is a way to give services externally-reachable urls. Each Ingress is a
-// collection of rules that allow inbound connections to reach the endpoints defined by
-// a backend.
+// An Ingress is a way to give services externally-reachable urls. Each Ingress
+// is a collection of rules that allow inbound connections to reach the endpoints
+// defined by a backend.
 type Ingress struct {
```

Comment:
If the former, please make the comment clearer. If the latter, this should be called "Egress".